interface/themes/oe-common/README-oe-common.txt

```diff
@@ -0,0 +1,5 @@
+Contains scss that is shared between all themes
+These rules generally apply to either individual pages or are oe- rules to be used anywhere in the application
+Here because they don't fit the category of 'core' component
+Can be used flexibly by importing individual scss files or by importing a single scss file called all-common-import.scss that has all the individual scss files
+REMEMBER!! for all-common-import.scss to do what it says, each time a scss file is added to this directory an @ import <file> line has to be added to all-common-import.scss
```

Comment:
@d3sandoval , Just checking to make sure the strategy above is in line with what you are trying to get us to do. -brady

Reply:
See it work in light, Manila where the whole lot is imported and the flat style sheet where individual files are imported 😀

Reply:
See it work in light, Manila where the whole lot is imported and the flat style sheet where individual files are imported 😀

Reply:
this should be a `.md` file so that github can parse it.